exchanges/interfaces.go

```diff
@@ -151,9 +152,11 @@ type CurrencyStateManagement interface {
 
 // AccountManagement defines functionality for exchange account management
 type AccountManagement interface {
```

Comment:
suggestion: Could have this interface in accounts?

Reply:
Could do, but not seeing a big benefit to setting a precedent like that here 🙂